### PR TITLE
CANS-1000: regression test stability fix

### PR DESCRIPTION
**Pull request by LeonidMarushevskyi**
## JIRA
https://osi-cwds.atlassian.net/browse/CANS-1000

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: spec/regression/case_worker_scenarios_spec.rb [hunk 1]
```diff
@@ -497,8 +497,8 @@ def click_save_and_summary_card_is_shown
   def warning_and_summary_card_shown_after_complete_button_clicked(is_reassessment)
     expect(@form.header.authorization_radio_no.checked?).to be(true)
     @form.footer.complete_button.click
-    expect(@form.app_globals).to have_complete_warning_modal
     sleep(2)
+    expect(@form.app_globals).to have_complete_warning_modal
```
Comment (denys-davydov) [inline spec/regression/case_worker_scenarios_spec.rb]: `have_complete_warning_modal` already uses the default timeout so the `sleep(2)` line doesn't seem to be useful here

File: spec/regression/case_worker_scenarios_spec.rb [hunk 2]
```diff
@@ -508,6 +508,7 @@ def warning_and_summary_card_shown_after_complete_button_clicked(is_reassessment
     expect(@form.footer).to have_complete_button
     @form.footer.complete_button.click
     sleep(2)
+    expect(@form.app_globals).to have_complete_warning_modal
```
Comment (denys-davydov) [inline spec/regression/case_worker_scenarios_spec.rb]: The same as above, the `sleep(2)` line doesn't seem to be useful here

File: spec/regression/case_worker_scenarios_spec.rb (hunk 1 above)
Comment (LeonidMarushevskyi) [reply spec/regression/case_worker_scenarios_spec.rb]: Not sure why but sleep helps with modal windows. Tried without slips and test failing not able to click modal button.

**Comment by LeonidMarushevskyi** [comment]
> small suggestion:
> 
> > Waiting for an element to become visible
> > A method that gets added by calling element is the wait_until_<element_name>_visible method. Calling this method will cause the test to wait for Capybara's default wait time for the element to become visible. You can customise the wait time by supplying a number of seconds to wait in-line or configuring the default wait time.
> 
> `@home.wait_until_search_field_visible`
> 
> # or...
> `@home.wait_until_search_field_visible(wait: 10)`

Just was working on this change. Applied.

**Comment by LeonidMarushevskyi** [comment]
Jenkins retest this, please

**Comment by LeonidMarushevskyi** [comment]
Jenkins retest this please

**Pull request merged**